lib/jefferies_tube/capistrano/maintenance.rb

```diff
@@ -0,0 +1,22 @@
+namespace :maintenance do
+  desc "Enable maintenance: set MESSAGE='We should be back in approximately 2 hours'"
+  task :enable do
+    on roles(:web) do |host, user|
+      within current_path do
+        message = ENV["MESSAGE"] || "We should return shortly."
```

Comment:
Maybe put the word "maintenance" in the default message somewhere. And why not make the default message the same as in the description?